charts/cluster-registrator/templates/job.yaml

```diff
@@ -3,14 +3,14 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: nirmata
+  name: {{ .Values.namespace }}
 {{- end }}
 ---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: nirmata-cluster-registrator
-  namespace: nirmata
+  namespace: {{ .Values.namespace }}
```

Comment:
If we say all objects are to be created in namespace of helm install, we should just delete namespace from objects. This one as well as ones below.